spec/fixtures/composite-action.yml

```
@@ -0,0 +1,40 @@
+     description: "The aws-secret-access-key used to authenticate with AWS"
+   app-name:
+     required: false
+     description: "The app name used by the React app"
+     default: Demo
+   api-url:
+     required: true
+     description: "The URL of the Express app"
+runs:
+  using: "composite"
+  steps:
+    - uses: aws-actions/configure-aws-credentials@v1
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 5 but found 4

(indentation)